pkg/envoy/xds_server.go

```diff
@@ -1617,8 +1617,8 @@ func getDirectionNetworkPolicy(ep endpoint.EndpointUpdater, l4Policy policy.L4Po
 		switch l4.Protocol {
 		case api.ProtoTCP:
 			protocol = envoy_config_core.SocketAddress_TCP
-		case api.ProtoUDP, api.ProtoSCTP:
-			// UDP/SCTP rules not sent to Envoy for now.
+		default:
+			// Other protocol rules not sent to Envoy for now.
```

Comment:
At first i thought that this change no longer properly handles the case where the protocol is `ANY` - until i realized that at this point the CNP (and therefore protocol) already has been sanitized and validated. (-> e.g. `L7 rules can only apply to TCP (not ANY) except for DNS rules`).

Consequently,  l4policies with Protocol `ANY` never reach this code (?).

Probably this is worth to include in the comment for the default case?

Reply:
We will eventually have to worry about supporting "ANY", but that PR will include a comprehensive review of the `l4.Protocol` field across the codebase as it will be removed in favor of the `U8Proto` field (or the type will change to use the `u8proto` pkg).